src/torrent_info.cpp

```diff
@@ -1525,6 +1525,10 @@ namespace {
 			{ return lhs.tier < rhs.tier; });
 	}
 
+	void torrent_info::clear_trackers() {
```

Comment:
please put the open brace on the next line

Reply:
Ok